src/app/components/search/search.component.html

```diff
@@ -1,48 +1,90 @@
 <!-- Start of form -->
 <form [formGroup]="form" (ngSubmit)="onSubmit()">
-  <div class="govuk-form-group">
-    <h1 class="govuk-label-wrapper">
-      <label class="govuk-label govuk-label--l" for="case-search"> Search for a case </label>
-    </h1>
-    <label class="govuk-label" for="case-search"> Case ID </label>
-    <div id="case-search-hint" class="govuk-hint">
+  <h1 class="govuk-label-wrapper">
+    <label class="govuk-label govuk-label--l" for="case-search"> Search for a case </label>
+  </h1>
+
+  <div *ngIf="errorSummary.length > 0" class="govuk-error-summary" data-module="govuk-error-summary">
+    <div role="alert">
+      <h2 class="govuk-error-summary__title">There is a problem</h2>
+      <div class="govuk-error-summary__body">
+        <ul class="govuk-list govuk-error-summary__list">
+          <li *ngFor="let error of errorSummary">
+            <a href="search#{{ error.fieldId }}">{{ error.message }}</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
+
+  <div class="govuk-form-group" [ngClass]="{ 'govuk-form-group--error': isSubmitted && f.case_number.errors }">
+    <label class="govuk-label" for="case_number"> Case ID </label>
+    <div id="case_number-hint" class="govuk-hint">
       Also known as a case reference or court reference. There should be no spaces.
     </div>
+    <div *ngIf="isSubmitted && f.case_number.errors">
+      <p *ngFor="let error of getFieldErrorMessages('case_number')" id="casenumber-error" class="govuk-error-message">
```

Comment:
Should we use a pipe here? (can be changed later)

Reply:
Yes I think we could refactor this to be it's own component potentially so we can pop it under all fields that have validation messages

Reply:
Resolving as this can come under another PR / ticket